main.go

```diff
@@ -83,14 +83,14 @@ func logic(ctx context.Context, o opts) error {
 	gusCfg.BasePath = gusBasePath
 	gusCli := gusapi.NewAPIClient(gusCfg)
 
-	if o.skipWaiting {
-		log.Print("skipping waiting, performing an immediate updateProcess")
-		if err := updateProcess(ctx, gusCli, machineID, o.gusServer, sbomHash, o.destinationDir, httpPassword, httpPort); err != nil {
-			// If the updateProcess fails we exit with an error
-			// so that gokrazy supervisor will restart the process.
-			return fmt.Errorf("error performing updateProcess: %v", err)
-		}
+	log.Print("Performing immediate initial updateProcess")
+	if err := updateProcess(ctx, gusCli, machineID, o.gusServer, sbomHash, o.destinationDir, httpPassword, httpPort); err != nil {
+		// If the updateProcess fails we exit with an error
+		// so that gokrazy supervisor will restart the process.
+		return fmt.Errorf("error performing updateProcess: %v", err)
```

Comment:
Could you wrap the err `%w` while you are at it? Thanks!

Reply:
done